_includes/navbar.html

```diff
@@ -25,4 +25,17 @@
       
       {% endfor %}
   </div>
+  <div class="theme-select">
+    <div class="nav-items">
+      <input type="radio" name="theme" id="theme-dark" value="dark"/>
+      <input type="radio" name="theme" id="theme-light" value="light"checked/>
```

Comment:
I think this should maybe start unchecked and then the JS checks one or the other? I noticed when I was in dark mode navigating between pages, the checked styling would jump from the light to dark option.

Reply:
@mhum removed the attribute, handling this more betterer in JS